lib/gengo-ruby/api_handler.rb

```diff
@@ -415,6 +415,23 @@ def deleteTranslationOrder(params = {})
       self.get_from_gengo('translate/order/:id'.gsub(':id', params.delete(:id).to_s), params)
     end
 
+    # Get all comments (the history) from a given order.
+    #
+    # Requirements:
+    # <tt>order_id</tt> - The ID of the order to get comments for.
+    def get_translation_order_comments(order_id)
+      get_from_gengo("translate/order/#{order_id}/comments")
+    end
+
+    # Post a comment for a translator or Gengo on an order.
+    #
+    # Options:
+    # <tt>id</tt> - The ID of the order you're commenting on.
+    # <tt>comment</tt> - The comment to put on the order.
+    def post_translation_order_comment(params = {})
+      send_to_gengo('translate/order/:id/comment'.gsub(':id', params.delete(:id).to_s), params)
```

Comment:
You don't have to use `gsub` but `sub`.
Also string interpolation is more common in Ruby as I commented in another place.

But we'll fix this issue in another PR since we have to merge this branch earlier.